src/rabbitmq.js

```diff
@@ -271,15 +271,14 @@ class RabbitMQ {
     return Promise.try(() => {
       const queueName = `${this.name}.${queue}`
       const bufferContent = this._validatePublish(queue, content, 'tasks')
-      const jobOpts = {
-        headers: {
-          publisher: this.name
-        }
+      const jobMeta = {
+        appId: this.name,
+        timestamp: Date.now()
```

Comment:
I made it timestamp and called it `timestamp` because this property is part of AMQP spec:

> timestamp (positive number): a timestamp for the message

See http://www.squaremobius.net/amqp.node/channel_api.html
There are few properties that are optional but defined by spec. `appId` is one of them, `timestamp` is another.
We can add our own properties if we want but it seems reasonable to me to reuse existing ones.

Also converting from timestamp to ISO would be easy.